sqlstore/src/main/resources/db/migration/V8123__SequencerPartitionContainer.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE SequencerPartitionContainer ADD COLUMN sequencingParametersId bigint,
+ADD CONSTRAINT `FK_Spc_Sp` FOREIGN KEY
```

Comment:
Please name this similar to other recently added foreign keys in MISO: `fk_sequencerPartitionContainer_sequencingParameters`